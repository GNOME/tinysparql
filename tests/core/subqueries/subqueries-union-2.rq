SELECT ?msg (
	SELECT ?date
	WHERE
	{
		{
			?msg example:date ?date .
		} UNION {
			?msg example:date ?date .
		}
	}
) AS ?date
WHERE {
	?msg a example:Message
} ORDER BY ?msg
