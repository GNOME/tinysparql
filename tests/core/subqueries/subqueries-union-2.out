"http://example.org/ns#msg1"	"2009-12-01T00:00:00Z"
"http://example.org/ns#msg2"	"2009-12-02T00:00:00Z"
"http://example.org/ns#msg3"	"2009-12-03T00:00:00Z"
"http://example.org/ns#msg4"	"2009-12-04T00:00:00Z"
"http://example.org/ns#msg5"	"2009-12-05T00:00:00Z"
